### PR TITLE
PYI-179: Add Onfido WebSDK

### PR DESCRIPTION
**Pull request by merlinc**
### What changed

This PR uses the Onfido Web UI to upload documents. It also allows device handoff during the journey.

The UI is only used for document upload, the checking process is handled by the existing Onfido API calls.

### Issue tracking

- [PYI-179](https://govukverify.atlassian.net/browse/PYI-179)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: views/onfido/onfido-display-web-sdk.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<div id='onfido-mount'></div>
+
+<!-- This should absolutely be locally included via our buiild pipeline -->
+<script src="https://assets.onfido.com/web-sdk-releases/6.14.0/onfido.min.js"></script>
+<link href="https://assets.onfido.com/web-sdk-releases/6.14.0/style.css" rel="stylesheet" type="text/css" />
+
+<script>
+Onfido.init({
+token: '{{ values.onfidoSDKToken }}',
```
Comment (martin-bucinskas) [inline views/onfido/onfido-display-web-sdk.html:14]: Note for beta, have a linter lint everything.
I assume this is Onfido implementation requirements? Not exactly sure what the token here is for, or what this method does, but having tokens on client side is scary, need to make sure that we can't do much with this token.

**Review by martin-bucinskas — DISMISSED** [review]

File: views/onfido/onfido-display-web-sdk.html (hunk 1 above)
Comment (merlinc) [reply views/onfido/onfido-display-web-sdk.html:14]: > Note for beta, have a linter lint everything.

Yeah, we should have a pre-push commit hook running some linting as well as any IDE linting to make sure we catch everything.

Also... not be putting JavaScript inside Nunjucks templates! 😂 

> I assume this is Onfido implementation requirements? Not exactly sure what the token here is for, or what this method does, but having tokens on client side is scary, need to make sure that we can't do much with this token.

This token is a per user token created in the `onfido-create-web-token` method - it's mapped 1:1 to the applicant, and is only able to upload documents/selfies.

> SDK tokens can only be used in our input-capture SDKs. They are restricted to an individual application, and expire after 90 minutes.

https://documentation.onfido.com/#generate-sdk-token

File: views/onfido/onfido-display-web-sdk.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{% extends "form-template.njk" %}
+{% set hmpoPageKey = "onfidoDisplayWebSDK" %}
+
+{% block mainContent %}
+
+<div id='onfido-mount'></div>
+
+<!-- This should absolutely be locally included via our buiild pipeline -->
+<script src="https://assets.onfido.com/web-sdk-releases/6.14.0/onfido.min.js"></script>
```
Comment (merlinc) [inline views/onfido/onfido-display-web-sdk.html:9]: Using this cdn version seems to also pull in some trackers. Will investigate with a compiled version in our build pipeline.

File: views/onfido/onfido-display-web-sdk.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+<link href="https://assets.onfido.com/web-sdk-releases/6.14.0/style.css" rel="stylesheet" type="text/css" />
+
+<script>
+Onfido.init({
+token: '{{ values.onfidoSDKToken }}',
+    containerId: 'onfido-mount',
+    onComplete: function(data) {
```
Comment (merlinc) [inline views/onfido/onfido-display-web-sdk.html:16]: This method doesn't seem to be firing atm - I wonder if its related to an error occurring in another method on submission? To investigate.

**Review by kerrr — DISMISSED** [review]

**Review by kerrr — APPROVED** [review]

**Pull request merged**